frontend/packages/noobaa-storage-plugin/integration-tests/mocks/deploymentData.ts

```diff
@@ -0,0 +1,37 @@
+export const testDeployment = 'test-deployment';
+
+export const deployment = {
+  apiVersion: 'apps/v1',
+  kind: 'Deployment',
+  metadata: {
+    name: testDeployment,
+  },
+  spec: {
+    selector: {
+      matchLabels: {
+        app: 'hello-openshift',
```

Comment:
Instead of `hello-openshift`, `test-openshift` might be better word to use here.

Reply:
Its an image name, can't be changed. @bipuladh you can mark this as resolved.